core/esmf-aspect-meta-model-python/tests/test_vocabulary.py

```diff
@@ -35,63 +35,63 @@ def assert_object_fields_value(unit, unit_fields_set: Set[Optional[str]]):
 
 
 # Unit
-def test_bamm_unit_implement_namespace() -> None:
-    unit = UNIT(BAMM_VERSION)
+def test_samm_unit_implement_namespace() -> None:
+    unit = UNIT(SAMM_VERSION)
     assert issubclass(type(unit), Namespace)
 
 
-def test_bamm_unit_fields() -> None:
+def test_samm_unit_fields() -> None:
     unit_fields_value: Set[Optional[str]] = set()
-    unit = UNIT(BAMM_VERSION)
+    unit = UNIT(SAMM_VERSION)
     assert_object_fields_value(unit, unit_fields_value)
 
 
-def test_bamm_unit_get_urn() -> None:
-    unit = UNIT(BAMM_VERSION)
-    uri_ref = unit.get_urn(BAMM.referenceUnit)
+def test_samm_unit_get_urn() -> None:
+    unit = UNIT(SAMM_VERSION)
+    uri_ref = unit.get_urn(SAMM.referenceUnit)
     assert issubclass(type(uri_ref), URIRef)
     assert uri_ref.toPython() == UNIT_URN
 
 
-def test_bamm_unit_get_type() -> None:
-    unit = UNIT(BAMM_VERSION)
+def test_samm_unit_get_type() -> None:
+    unit = UNIT(SAMM_VERSION)
     string_type = unit.get_name(UNIT_URN)
     assert string_type == "referenceUnit"
 
 
-# Bamme
-def test_bamm_bamme_implement_namespace() -> None:
-    bamme = BAMME(BAMM_VERSION)
-    assert issubclass(type(bamme), Namespace)
+# Samme
+def test_samm_samme_implement_namespace() -> None:
+    samme = SAMME(SAMM_VERSION)
+    assert issubclass(type(samme), Namespace)
 
 
-def test_bamm_bamme_fields() -> None:
-    bamme_fields_value: Set[Optional[str]] = {"TimeSeriesEntity", "ThreeDimensionalPosition", "timestamp", "x", "y", "z", "value"}
-    bamme = BAMME(BAMM_VERSION)
-    assert_object_fields_value(bamme, bamme_fields_value)
+def test_samm_samme_fields() -> None:
+    samme_fields_value: Set[Optional[str]] = {"TimeSeriesEntity", "ThreeDimensionalPosition", "timestamp", "x", "y", "z", "value"}
```

Comment:
This is not called `ThreeDimensionalPosition` any more (since a long time), but `Point3d`. I don't know all the places where this needs to be fixed however.